test/system/groups/group_links_test.rb

```diff
@@ -208,5 +211,66 @@ def setup
 
       assert_text 'Resource not found'
     end
+
+    test 'group member of Group C can access Group B as it is shared with Group C' do
+      login_as users(:user25)
+
+      namespace_group_link = namespace_group_links(:namespace_group_link9)
+
+      visit group_url(namespace_group_link.namespace)
+
+      assert_no_text I18n.t(:'action_policy.policy.group.read?',
+                            name: namespace_group_link.namespace.name)
+
+      assert_selector 'h1', text: namespace_group_link.namespace.human_name, count: 1
+      assert_selector 'p', text: namespace_group_link.namespace.description, count: 1
+    end
+
+    test 'group member of Group B can access Group A as Group B is shared with Group A' do
```

Comment:
**nitpick (blocking):** Shouldn't this be written as `group member of Group B can access Group A as it is shared with Group B`

Reply:
Updated in 038f17b9ee10a2609acd0c521ba6e415362e9ca2